README.md

```diff
@@ -11,8 +11,10 @@ The module is designed to be instantiated many times, once for each desired land
 This is currently split logically into the following capabilities:
 
 - Subscription creation and management group placement
-- Hub & spoke networking
-- Virtual WAN networking
+- Networking - deploy multiple vnets with:
+  - Hub & spoke connectivity
+  - vWAN connectivity
+  - Mesh peering (peering between spokes)
```

Comment:
Question: can you include hubs as part of the mesh? Should we link to the vnet peering limits?